tests/integration.py

```diff
@@ -0,0 +1,128 @@
+"""
+Integration tests for link022 gnmi functions.
```

Comment:
Maybe create a README file explain the workflow and python library dependencies.

Reply:
Added a README file. Thanks!